package.json

```diff
@@ -319,7 +319,7 @@
     "tar": "^7.4.0",
     "tar-fs": "^3.0.6",
     "terser": "^5.31.2",
-    "tsx": "4.11.2",
+    "tsx": "4.17.0",
```

Comment:
@privatenumber our ci has been broken since `tsx@4.12.x`.

Reply:
Thanks, can you provide a minimal reproduction to the tsx repo?